diff

This fixes a couple IDL generation problems that would break SciMoz builds

Index: /Users/shanec/main/Apps/Komodo-devel/contrib/scintilla/include/Scintilla.iface
===================================================================
RCS file: /cvsroot/scintilla/scintilla/include/Scintilla.iface,v
retrieving revision 1.307
diff -u -r1.307 include/Scintilla.iface
--- include/Scintilla.iface	23 Apr 2007 02:26:05 -0000	1.307
+++ include/Scintilla.iface	24 May 2007 01:36:19 -0000
@@ -1559,22 +1559,22 @@
 get int GetPrintWrapMode=2407(,)
 
 # Set a fore colour for active hotspots.
-set void SetHotspotActiveFore=2410(bool useSetting, colour fore)
+fun void SetHotspotActiveFore=2410(bool useSetting, colour fore)
 
 # Get the fore colour for active hotspots.
-get colour GetHotspotActiveFore=2494(,)
+fun colour GetHotspotActiveFore=2494(,)
 
 # Set a back colour for active hotspots.
-set void SetHotspotActiveBack=2411(bool useSetting, colour back)
+fun void SetHotspotActiveBack=2411(bool useSetting, colour back)
 
 # Get the back colour for active hotspots.
-get colour GetHotspotActiveBack=2495(,)
+fun colour GetHotspotActiveBack=2495(,)
 
 # Enable / Disable underlining active hotspots.
-set void SetHotspotActiveUnderline=2412(bool underline,)
+fun void SetHotspotActiveUnderline=2412(bool underline,)
 
 # Get whether underlining for active hotspots.
-get bool GetHotspotActiveUnderline=2496(,)
+fun bool GetHotspotActiveUnderline=2496(,)
 
 # Limit hotspots to single line so hotspots on two lines don't merge.
 set void SetHotspotSingleLine=2421(bool singleLine,)
